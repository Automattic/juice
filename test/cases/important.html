<html><body>
<a id="a" style="color: blue; background: green;">woot</a>
</body></html>
