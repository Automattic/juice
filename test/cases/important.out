<html><body>
<a id="a" style="color: red; background: blue; border-style: dashed;">woot</a>
</body></html>
